packages/protocol/scripts/truffle/verify-bytecode.ts

```diff
@@ -47,7 +47,7 @@ module.exports = async (callback: (error?: any) => number) => {
     const version = getReleaseVersion(branch)
 
     const registry = await Registry.at(celoRegistryAddress)
-    const buildArtifacts = getBuildArtifacts(artifactsDirectory)
+    const buildArtifacts = getBuildArtifacts(artifactsDirectory) // TODO this should also include 0.8 contracts
```

Comment:
I think this TODO is not relevant anymore as the artifacts folder is now iterated